tests/unit/amo/components/TestCollection.js

```diff
@@ -1231,11 +1231,16 @@ describe(__filename, () => {
     const addonId = addons[0].addon.id;
     const detail = createFakeCollectionDetail({
       authorId: authorUserId,
+      count: 30,
```

Comment:
I'd add a comment saying that we pretend there are > 25 addons so that we are on page 2.

Reply:
what about doing something like this: 

```javascript
 // This will simulate a few items on the 2nd page.
 count: DEFAULT_API_PAGE_SIZE + 2,
```

Reply:
Yes!

Reply:
lol..

but I just realized this is actually identical to the test I have below now. I I guess I can remove this